### PR TITLE
Controlgroup: Small fixes to entry

### PR DESCRIPTION
**Pull request by agcolom**

### CODE REVIEW

File: entries/controlgroup.xml [hunk 1]
```diff
@@ -31,15 +31,15 @@
...
+				<p>By default, controlgroup displays its controls in a horizontal layout. Use this option to use a vertical layout instead.</p>
 			</desc>
 			<type name="String" />
 		</option>
 		<xi:include href="../includes/widget-option-disabled.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 		<option name="onlyVisible" default="true" example-value="false">
-			<desc>Sets whether to exclude invisible children in the assignment of rounded corners. When set to false, all children of a controlgroup are taken into account when assigning rounded corners, including hidden children. Thus, if, for example, the controlgroup's first child is hidden, the controlgroup will, in effect, not have rounded corners on the top edge.</desc>
+			<desc>Sets whether to exclude invisible children in the assignment of rounded corners. When set to false, all children of a controlgroup are taken into account when assigning rounded corners, including hidden children. Thus, if, for example, the controlgroup's first child is hidden, the controlgroup will, in effect, not have rounded corners on the left edge.</desc>
```
Comment (jzaefferer) [inline entries/controlgroup.xml]: Makes sense for the default `direction`. Could be made explicit though, something like "not have rounded corners on the left edge (for `direction: "horizontal"`)."

Comment (jzaefferer) [inline entries/controlgroup.xml]: Makes sense for the default `direction`. Could be made explicit though, something like "not have rounded corners on the left edge (for `direction: "horizontal"`)."

Comment (agcolom) [inline entries/controlgroup.xml]: how about this (Added text in bold): "Sets whether to exclude invisible children in the assignment of rounded corners. When set to false, all children of a controlgroup are taken into account when assigning rounded corners, including hidden children. Thus, if, for example, the controlgroup's first child is hidden **and the default horizontal layout is applied**, the controlgroup will, in effect, not have rounded corners on the left edge."

Comment (agcolom) [inline entries/controlgroup.xml]: We could also have an additional sentence after this to say: "Likewise, if the controlgroup has a vertical layout and its first child is hidden, the controlgroup will not have rounded corners on the top edge."

Comment (jzaefferer) [inline entries/controlgroup.xml]: Both additions sound good to me.

**Comment by scottgonzalez** [comment]
@agcolom Can you push the updates discussed above? Thanks.

**Comment by agcolom** [comment]
@scottgonzalez done :-)

**Comment by scottgonzalez** [comment]
Merged into 1-12.

**Pull request closed without merging**